guides/cheat-sheets/api.cheatmd

```diff
@@ -0,0 +1,139 @@
+# API Usage
+
+This cheat sheet covers basic use of the Discord API through the `Nostrum.Api` module.
+
+## Messages
+{: .col-2}
+
+### Sending a message
+
+```elixir
+utc_now = DateTime.now!("Etc/UTC")
```

Comment:
Small nitpic, but you could just use [DateTime.utc_now/1](https://hexdocs.pm/elixir/DateTime.html#utc_now/1) here

Reply:
Yes, I agree. We shouldn't have to parse a string every time here

Reply:
Fixed in 56649d4da88c6ed0d537402674bc424a40ccfc64